### ISSUE TITLE
retrieve_test_operation_failure doesn't work with Xcode 11.3.1

### ISSUE THREAD

**Issue by Cyberbeni** [lyndsey-ferguson/fastlane-plugin-test_center#202]
### New Issue Checklist

- [x] Updated `fastlane-plugin-test_center` to the latest version
- [x] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [x] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [x] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

### Issue Description
Related to #180 
Test failure:
```
[0;36;49m[worker 2] [0mINFO [2020-03-01 14:36:01.57]: ▸ [0;35;49mTesting failed:[0m
[0;36;49m[worker 2] [0mINFO [2020-03-01 14:36:01.58]: ▸ [0;35;49m	BugreportUITests:[0m
[0;36;49m[worker 2] [0mINFO [2020-03-01 14:36:01.58]: ▸ [0;35;49m		BugreportUITests-Runner.app (3098) encountered an error (Early unexpected exit, operation never finished bootstrapping - no restart will be attempted. (Underlying error: Test runner exited before starting test execution.))[0m
```
Related pattern match can't extract the failure reason:
https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/c4cf9483f5a3fe7278b76d270490e828e22d391e/lib/fastlane/plugin/test_center/helper/multi_scan_manager/retrying_scan_helper.rb#L270

### Complete output when running fastlane, including the stack trace and command used
Included in #180 

### Environment
Xcode 11.3.1

**Comment by lyndsey-ferguson** [comment]
@Cyberbeni what is your expected behavior in this case? I think the generic error stating `Underlying error: Test runner exited before starting test execution` could apply to anything. I think it would be better if I were to look for this error, and then exit: `Please unlock your device and reattach`

**Comment by Cyberbeni** [comment]
I think the goal is to fix known fixable errors, exit on known unfixable ones and just retry everything else.
In my case, I only run tests on simulator and I only get the occasional random error that I want to retry.

**Comment by lyndsey-ferguson** [comment]
Hm. but this particular log indicated that it failed because the device was unlocked. In this case, that is something that requires an early exit. Sometimes a `Test runner exited before starting test execution` could just be an intermittent error and a retry would work...

**Comment by Cyberbeni** [comment]
?

The point is `test_operation_failure_match = /Test operation failure: (?<test_operation_failure>.*)$/ =~ test_session_last_messages` doesn't match anything because the output is different.

**Comment by lyndsey-ferguson** [comment]
Ah, so your suggesting that `Test operation failure:` is no longer going to happen? You're saying that as a general "catch" this will no longer work?

**Comment by Cyberbeni** [comment]
Yes.

**Comment by lyndsey-ferguson** [comment]
I need the sad emoji.

**Comment by lyndsey-ferguson** [comment]
I'll double check this by running a parallel run with more sims than it should be able to handle. If the pattern has changed, I should no longer see "test operation failure"

**Comment by lyndsey-ferguson** [comment]
Yes, the errors have changed. 
I see this: 
```
Unexpected timeout after sending message com.appian.tempo.ipc.message.test-command`
```
```
Connection peer refused channel request for "dtxproxy:XCTestManager_IDEInterface:XCTestManager_DaemonConnectionInterface
```

**Comment by lyndsey-ferguson** [comment]
Can you modify your `Pluginfile` per my instructions below, run `bundle install`, and then run your fastlane again (with the `--verbose` flag)?

Pluginfile:
```
gem 'fastlane-plugin-test_center',  '3.9.1.b.1'
```

If there are still problems, please let me know and attach the console output as a text file to this issue (makes it easier for me to review).  If it works, please let me know.

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.10.2](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.10.2)
